src/components/implementation/tests/micro_booter/mb_tests.c

```diff
@@ -925,15 +876,35 @@ test_run_mb(void)
  * TODO: Fix those eventually.
  */
 void
+__block_vm(void)
```

Comment:
Yeah, I think it's my habit to prefix `__` for a function that's private (like `static`) but keep forgetting to make it `static` at the very least.